scripts/create_test_subset.py

```diff
@@ -156,7 +156,7 @@ def main(
     except exceptions.ApiException:
         seq_info_by_s_id = {}
     else:
-        seq_info_by_s_id = dict(zip(sample_ids, seq_infos))
+        seq_info_by_s_id = {seq['sample_id']: seq for seq in seq_infos}
```

Comment:
Oh damn, this burned production-pipelines early on too.

Reply:
Yeah, the second I saw it, I remembered the last incident. I tried to search the code base to see if we do this anywhere else and didn't notice anything.